cockpit/spec/dbus/dbus.test.js

```diff
@@ -0,0 +1,11 @@
+const cockpit = require("cockpit");
+const service = cockpit.dbus("com.redhat.RHSM1", {"superuser": "require"})
+
+describe("Check that we can Register with subscription-manager", function() {
+
+    it("Should create a unix socket when we register", function() {
+        let proxy = service.proxy("com.redhat.RHSM1.RegisterServer", "/com/redhat/RHSM1/RegisterServer");
+        let start = proxy.Start();
```

Comment:
I discovered a problem with this code when testing.  It should actually be something like this:

```javascript
    let proxy = service.proxy("com.redhat.RHSM1.RegisterServer", "/com/redhat/RHSM1/RegisterServer");
    function handler(result) {
        console.log(result);
        return result;
    }

    function err_handler(err) {
        console.log(err);
    }

    var registerSock = "";
    proxy.wait(()=> {
        let start = proxy.Start();

        start.then(handler)
            .then(sock => {
                console.log(sock);
                registerSock = sock;
            })
            .fail(err_handler);
    })

    return registerSock;
```